packages/wallet/backend/src/createContainer.ts

```diff
@@ -265,10 +265,12 @@ export const createContainer = (config: Env): Container<Bindings> => {
     const logger = await container.resolve('logger')
     const rafikiClient = await container.resolve('rafikiClient')
     const transactionService = await container.resolve('transactionService')
+    const rapydService = await container.resolve('rapydService')
```

Comment:
Why are we using `rapydService` here?

Reply:
because that is where the daily rates retrieval is done.
Rico had it inside the Rapyd module in his PR, and since we didn't have a better place for it yet, I was ok with it.
We could extract it somewhere else in the future I guess.

Reply:
The two options would be a separate module for rates, or a ratesService inside QuoteService.
I'd go for the separate module, since rates are also going to be used in Exchange, not just in Quotes.
@rico191013 ?

Reply:
moved it into a separate module, please have a look